### ISSUE TITLE
Class 'Omnipay\Common\Helper' not found

### ISSUE THREAD

**Issue by bryantwells** [craftcms/commerce#344]
When submitting payment method using a basic form and dummy gateway, an error is thrown:

```
<form method="POST" class="form-horizontal" id="paymentForm">
    <input type="hidden" name="action" value="commerce/payments/pay"/>
    <input type="hidden" name="redirect" value="/shop/customer/order?number={number}"/>
    <input type="hidden" name="cancelUrl" value="/shop/checkout/payment"/>

    {{ csrfInput() }}
    {{ cart.gateway.getPaymentFormHtml({})|raw }}

    <button class="button button-primary" type="submit">Pay Now</button>
</form>
```

```
Class 'Omnipay\Common\Helper' not found

1. in [...]/craftcms/commerce/src/models/payments/CreditCardPaymentForm.php at line 117
```

### Additional info

- Craft version: Craft Solo 3.0.7
- PHP version: 7.1.12
- Database driver & version: MySQL 5.6.38
- Plugins & versions: Commerce 2.0.0-beta.4.1

**Comment by bryantwells** [comment]
couldn't find this documented anywhere — but running `composer require craftcms/commerce-omnipay` did the trick

**Comment by lukeholder** [comment]
This has been fixed in b56d4cd79388d4, will be in the next release. There should not be a need to require the omnipay package for commerce.
